src/components/Select/__tests__/Select.base-actions.test.tsx

```diff
@@ -40,6 +42,15 @@ describe('Select base actions', () => {
             await user.keyboard('[Space]');
             getByTestId(SelectQa.POPUP);
         });
+
+        test('default with defaultOpen prop', async () => {
+            await act(async () => {
+                setup({defaultOpen: true});
+            });
+            const selectControl = screen.getByTestId(TEST_QA);
+            expect(selectControl).toHaveClass(SELECT_CONTROL_OPEN_CLASS);
+            screen.getByTestId(SelectQa.POPUP);
+        });
```

Comment:
Could you add test without using `defaultOpen` property?

Reply:
What do you mean? Add a test with checking that popup doesn't appear and Select stays "closed" while rendering without `defaultOpen`  property?